### DISCUSSION TITLE
Livewire and Multiples Server Requests?

### DISCUSSION THREAD

**Discussion by devajmeireles** [livewire/livewire#4383] [4. General]
Hello community!

I would like to clarify a question regarding Livewire.

I participate in many communities about Laravel and in many of them I see the same topic being raised over and over again: "Multiple Livewire Requests".

In these discussions, the subjects covered are the numerous requests that Livewire makes to the server, and that because of that there are certain problems with the application's performance.

Of course, someone like me, who has been working with PHP for over 8 years, knows that a tool that is misused will cause problems every time, and it is a great truth that, unfortunately, many developers do not know how to use such tools and condemn them, however considering these facts I would like to know about the topic.

From the little I explored about Livewire I understood that the databind it offers requires several requests on the server to fill variables, for that we could use the debouce methods and others, but speaking of large applications, how is this handled?

In the communities I'm part of, I've seen InertiaJS gain significant ground due to the way it handles things.  Many of my colleagues, for example, do not recommend using Livewire for large applications and I really never seen great applications based on Livewire! I don't know if it's a coincidence or not.

So... I wonder what you guys have to say about this overall?

**Comment by calebporzio** [comment]
Thanks for posting this @ajmeireles - can you expound on that "multiple requests" you're talking about?

What I'm hearing you say is:

Because data binding in livewire (done with `wire:model`) sends many requests as a user fills out a form, it makes the app slower and has a higher server demand.

Is that right?

Or are you referring to other things? Thanks!

**Reply by devajmeireles** [reply]
Yea!  you're right.

Since I heard the first reports about this "problem" on Livewire about many requests I understood that this was the situation.  The "problem" was linked to the fact of the way it databinds, sending requests to the server.

I would like to hear your opinion on these topics.  How can we avoid such problems, would it be using the debounce approach everytime or are there other techniques?

Another thing I would like to know from you, because I understand that you are one of the project's maintainers, is if you know great applications that are based only on Livewire so that I could see the overall performance?

Thank you!
